### PR TITLE
Add org.electrum.electrum

### PR DESCRIPTION
**Pull request by A6GibKm**
Follow-up of #837. I can take care of updating this to newer releases.

Add @bauerj as a contributor please.

### CODE REVIEW

**Comment by A6GibKm** [comment]
128x128 or svg icons are not provided at upstream yet.

**Comment by A6GibKm** [comment]
bot, build org.electrum.electrum

**Comment by A6GibKm** [comment]
bot, build org.electrum.electrum

File: org.electrum.electrum.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.electrum.electrum</id>
+  <name>Electrum</name>
+  <developer_name>spesmilo</developer_name>
+  <launchable type="desktop-id">org.spesmilo.Electrum.desktop</launchable>
```
Comment (nedrichards) [inline org.electrum.electrum.appdata.xml]: this id doesn't match. should it be `org.electrum.electrum.desktop`?

Comment (A6GibKm) [reply org.electrum.electrum.appdata.xml]: Yes, you are correct.

**Comment by LeandroStanger** [comment]
bot, build org.electrum.electrum

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.electrum.electrum

**Pull request closed without merging**